draft-schwartz-add-ddr-forwarders.md

```diff
@@ -49,175 +49,192 @@ informative:
 
-# Security Considerations {#security-considerations}
+# Management of local blocking functionality
 
-When the client uses the conservative validation policy described in {{DDR}}, and a DDR-enabled resolver is identified by a private IP address, the client can establish a secure DDR connection only in the absence of an active attacker.  An on-path attacker can impersonate the resolver and intercept all queries, by preventing the DDR upgrade or advertising their own DDR endpoint.
+Certain local DNS forwarders block access to domains associated with malware and other threats. Others block based on the category of service provided by those domains, e.g. domains hosting services that are not appropriate for a work or school environment. In the short term to ensure this service is not lost due to a cross-forwarder upgrade, the maintainers can simply add "resolver.arpa" to their actively curated list of domains to block. This pattern has been deployed by Mozilla, with the domain "use-application-dns.net" {{MOZILLA-CANARY}}.
 
-These basic security properties also apply if the client uses the relaxed validation policy described in {{client-policy}}.  Nonetheless, there are some subtle but important differences in the security properties of these two policies.
+In the long term, it is best for filtering providers to implement support for encrypted DNS. The following subsections describe some ways to implement this.
 
-## Transient attackers
+## Local implementation with DNR
 
-With the conservative validation policy, a transient on-path attacker can only intercept queries for the duration of their active presence on the network, because the client will only send queries to the original (private) server IP address.
+The local forwarder can be upgraded to one that implements an encrypted DNS service discoverable through DNR. This requires a TLS certificate on the local device, proving ownership of the chosen ADN. Onward queries to the internet should also be protected with encryption.
 
-With the relaxed validation behavior, a transient on-path attacker could implant a long-lived DDR response in the client's cache, directing its queries to an attacker-controlled server on the public internet.  This would allow the attack to continue long after the attacker has left the network.
+## Local implementation with DDR
 
-Solving or mitigating this attack is of great importance for the user's security.
+If the local forwarder can be assigned a public IP address, and it can be upgraded to offer an encrypted DNS service, this can then be made discoverable through classic DDR. This requires a TLS certificate on the local device, proving ownership of the public IP. Onward queries to the internet should also be protected with encryption.
```

Comment:
```suggestion
If the local forwarder can be upgraded to offer an encrypted DNS service, this can then be made discoverable through classic DDR. If the device has a private IP (as presumed for RVS), a self-signed certificate is sufficient. Onward queries to the internet SHOULD also be protected with encryption.
```